lib/shower/default-options.js

```diff
@@ -1,17 +1,13 @@
 export default {
     container_selector: '.shower',
-
-    debug_mode: false,
```

Comment:
Do wee need debug mode? It just sets a class and logs to console.

Reply:
I don’t really see the point. Maybe only from developer’s perspective. But I guess there are better tools for that.